actions/signing-event/action.yml

```diff
@@ -18,8 +18,10 @@ runs:
       run: |
         if tuf-on-ci-status >> status-output;  then
           echo "status=success" >> $GITHUB_OUTPUT
+          cat status-output >> "$GITHUB_STEP_SUMMARY"
         else
           echo "status=failure" >> $GITHUB_OUTPUT
+          echo "Failed to get status of event" >> "$GITHUB_STEP_SUMMARY"
```

Comment:
This is not correct I think, the status return code is a bit unintuitive:
* tuf-on-ci-status exit code 0 means the signing event is "complete" (ready to merge)
* tuf-on-ci-status other exit codes means the signing event is not complete

 (obviously the exit code is also non-zero for actual bugs: this is not a well designed part)
 
 My suggestion here might be to
 * unconditionally print the status-output (see two lines below)
 * maybe print the summary variable in the following file-issue task (I assume doing that in js is not hard)
 
 (or just file another issue to handle this case)

Reply:
I think it's a bit tricky to add a step summary in the `file-issue`, I can take a look at that next week.

Reply:
As it's a `uses: ` step. But I think it should be possible.

Reply:
Marking this a draft and see if I can fix this.

Reply:
:+1: Also fine to leave this part out of the PR and just file a new issue about it